book/website/communication/social-media/social-media-twitter-multiple.md

```diff
@@ -0,0 +1,38 @@
+(cm-social-media-twitter)=
+# Managing Multiple Twitter Accounts
+
+Managing multiple accounts on Twitter can be challenging and you might be following a certain social media policy or approach that is different from how you would tweet from your personal account.
+
+Many projects and organisations have their own Twitter account that is used for advertising their research outputs and events.
+
+(cm-social-media-twitter-organisation)=
+## Managing an Organisational Twitter Account
+
+**1. Be sure that you are clear about what the purpose of the account is and the type of tweets that the organisation wants.**
+
+* Is it going to be a professional tone or more relaxed and fun.
+* Are there certain restrictions about what content you can tweet about such as not sharing data or results?
+
+**2. Use scheduling to time your tweets**
+
+* This can be done through the Twitter web browser or by using a Twitter app such as Tweetdeck.
+* Tweetdeck is the only app owned by Twitter and it is free to use.
+But there are other apps for all different platforms, although you often have to pay for their services.
+See this [blog](https://www.reviewgeek.com/52119/the-best-twitter-apps-for-every-platform/) for more information.
+* There are other apps, such as [Buffer](https://buffer.com/), that you can use to control multiple social media channels.
+* Scheduling tweets is particularly helpful if you are running an event as you can schedule tweets well in advance to give important information.
+
+**3. Preventing miss tweeting**
+
+* When you are responsible for multiple tweeter accounts, there is the possibility of miss tweeting.
+This is when you tweet out, for example, a personal tweet using an organisational account.
+* Tweetdeck has a feature that allows you to add a confirmation step to perform an extra check before tweeting.
+This helps to prevent this issue.
+
+**4. Enabling multiple users of the same account**
+
+* Also on Tweetdeck, you can set up an account as a team to allow multiple users to have different access.
+* There will be one person that is the admin for the account and then the other users are contributors.
+* As a contributor you can post Tweets, Direct Message, like, and Retweet for the team account.
```

Comment:
post a Tweet or convert the rest into Direct Messages, likes, retweets so that the counts are the same